{"uuid":"3061afa8-7327-4244-9c0c-92357d27b773","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["a589e943-2958-4fd8-9f23-1505df325123"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378723936,"stop":1673378723937}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378723945,"stop":1673378723947},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378724454,"stop":1673378724454},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673378724620,"stop":1673378724620},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"387f29b7-fbf3-4337-8f52-3fdefa5b80d2-attachment","type":"image/png"},{"name":"Myscreenshot","source":"9fedf05c-9f9c-4175-b3e6-71f6e6955bde-attachment"}],"parameters":[],"start":1673378724641,"stop":1673378724991}],"start":1673378723936,"stop":1673378724991}